### PR TITLE
Pharmacy/Surgery - Change the IO painkiller checks and change the closed reduction checks

### PR DESCRIPTION
**Pull request by Cplhardcore**
**When merged this pull request will:**
- _Change the pain addition with an IO to only apply when a medication is at its strongest_
- _Add the missing pain meds (Fentanyl/Ketamine/Nalbuphine) to the check_
- _Change the Closed reduction check to be in line with the IO check_

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/pharma/functions/fnc_applyIV.sqf [hunk 1]
```diff
@@ -49,9 +49,12 @@ if (_usedItem isEqualTo "kat_IV_16") then {
...
 
-    private _count = [_patient, "Lidocaine"] call ACEFUNC(medical_status,getMedicationCount);
-    private _count2 = [_patient, "Morphine"] call ACEFUNC(medical_status,getMedicationCount);
-    if (_count == 0 && _count2 == 0) then {[_patient, 0.8] call ACEFUNC(medical_status,adjustPainLevel)};
+    private _count = [_patient, "Lidocaine", false] call ACEFUNC(medical_status,getMedicationCount);
+    private _count2 = [_patient, "Morphine", false] call ACEFUNC(medical_status,getMedicationCount);
+    private _count3 = [_patient, "Nalbuphine", false] call ACEFUNC(medical_status,getMedicationCount);
+    private _count4 = [_patient, "Fentanyl", false] call ACEFUNC(medical_status,getMedicationCount);
+    private _count5 = [_patient, "Ketamine", false] call ACEFUNC(medical_status,getMedicationCount);
+    if (_count <=  0.6 && _count2 <=  0.6 && _count3 <=  0.6 && _count4 <=  0.6 && _count5 <=  0.6) then {[_patient, 0.8] call ACEFUNC(medical_status,adjustPainLevel)};
```
Comment (BlueTheKing) [inline addons/pharma/functions/fnc_applyIV.sqf]:
```suggestion
    private _lidocaineCount = [_patient, "Lidocaine", false] call ACEFUNC(medical_status,getMedicationCount);
    private _morphineCount = [_patient, "Morphine", false] call ACEFUNC(medical_status,getMedicationCount);
    private _nalbuphineCount = [_patient, "Nalbuphine", false] call ACEFUNC(medical_status,getMedicationCount);
    private _fentanylCount = [_patient, "Fentanyl", false] call ACEFUNC(medical_status,getMedicationCount);
    private _ketamineCount = [_patient, "Ketamine", false] call ACEFUNC(medical_status,getMedicationCount);
    if (_lidocaineCount <= 0.6 && _morphineCount <= 0.6 && _nalbuphineCount <= 0.6 && _fentanylCount <= 0.6 && _ketamineCount <= 0.6) then {[_patient, 0.8] call ACEFUNC(medical_status,adjustPainLevel)};
```

File: addons/surgery/functions/fnc_closedReductionLocal.sqf [hunk 2]
```diff
@@ -22,10 +22,12 @@ params ["_medic", "_patient", "_bodyPart"];
...
-private _count1 = [_patient, "Lidocaine"] call ACEFUNC(medical_status,getMedicationCount);
-private _count2 = [_patient, "Morphine"] call ACEFUNC(medical_status,getMedicationCount);
-
-if (_count1 == 0 && _count2 == 0) then {
+private _count = [_patient, "Lidocaine", false] call ACEFUNC(medical_status,getMedicationCount);
+private _count2 = [_patient, "Morphine", false] call ACEFUNC(medical_status,getMedicationCount);
+private _count3 = [_patient, "Nalbuphine", false] call ACEFUNC(medical_status,getMedicationCount);
+private _count4 = [_patient, "Fentanyl", false] call ACEFUNC(medical_status,getMedicationCount);
+private _count5 = [_patient, "Ketamine", false] call ACEFUNC(medical_status,getMedicationCount);
+if (_count <=  0.8 && _count2 <=  0.8 && _count3 <=  0.8 && _count4 <=  0.8 && _count5 <=  0.8) then {
```
Comment (BlueTheKing) [inline addons/surgery/functions/fnc_closedReductionLocal.sqf]:
```suggestion
private _lidocaineCount = [_patient, "Lidocaine", false] call ACEFUNC(medical_status,getMedicationCount);
private _morphineCount = [_patient, "Morphine", false] call ACEFUNC(medical_status,getMedicationCount);
private _nalbuphineCount = [_patient, "Nalbuphine", false] call ACEFUNC(medical_status,getMedicationCount);
private _fentanylCount = [_patient, "Fentanyl", false] call ACEFUNC(medical_status,getMedicationCount);
private _ketamineCount = [_patient, "Ketamine", false] call ACEFUNC(medical_status,getMedicationCount);
if (_lidocaineCount <= 0.8 && _morphineCount <= 0.8 && _nalbuphineCount <= 0.8 && _fentanylCount <= 0.8 && _ketamineCount <= 0.8) then {
```

**Review by BlueTheKing — CHANGES_REQUESTED** [review]

File: addons/pharma/functions/fnc_applyIV.sqf (hunk 1 above)
Comment (Cplhardcore) [reply addons/pharma/functions/fnc_applyIV.sqf]: Fixed

File: addons/surgery/functions/fnc_closedReductionLocal.sqf (hunk 2 above)
Comment (Cplhardcore) [reply addons/surgery/functions/fnc_closedReductionLocal.sqf]: Fixed

**Review by BlueTheKing — APPROVED** [review]

**Review by MiszczuZPolski — APPROVED** [review]

**Pull request merged**